src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -366,15 +368,15 @@ private static boolean shouldBeOnSameLine(RightCurlyOption bracePolicy, Details
      *
      * @param bracePolicy option for placing the right curly brace
      * @param details Details for validation
-     * @param targetSrcLine A string with contents of rcurly's line
+     * @param targetSrcLine Unicode code points array with contents of rcurly's line
      * @return true if a right curly should be alone on a line.
      */
     private static boolean shouldBeAloneOnLine(RightCurlyOption bracePolicy,
                                                Details details,
-                                               String targetSrcLine) {
+                                               int... targetSrcLine) {
         return bracePolicy == RightCurlyOption.ALONE
                     && shouldBeAloneOnLineWithAloneOption(details, targetSrcLine)
-                || (bracePolicy == RightCurlyOption.ALONE_OR_SINGLELINE
+                    || (bracePolicy == RightCurlyOption.ALONE_OR_SINGLELINE
```

Comment:
Why change formatting here?

Reply:
Oh sorry!,

BTW, can you pls help me understand, reason for 
<pre>
       && shouldBeAloneOnLineWithAloneOption(details, targetSrcLine)
   || (bracePolicy == RightCurlyOption.ALONE_OR_SINGLELINE
       || details.shouldCheckLastRcurly)
</pre>
instead of
<pre> 
 && shouldBeAloneOnLineWithAloneOption(details, targetSrcLine)
 || (bracePolicy == RightCurlyOption.ALONE_OR_SINGLELINE
      || details.shouldCheckLastRcurly)
</pre>

Reply:
Your example is better, the existing formatting was probably just the default formatting of whatever IDE the author was using :) If you want to fix it, feel free, but it should be in a separate commit, like `minor: fix formatting of expression`.

Reply:
Resolved.